authors/apps/articles/admin.py

```diff
@@ -0,0 +1,4 @@
+from django.contrib import admin
+from .models import Article
```

Comment:
Ensure you separate local imports from standard library imports with one blank line